### PR TITLE
IRIS-4251 | Remove Achievements from Wiki Labs

### PR DESCRIPTION
**Pull request by slayful**
https://wikia-inc.atlassian.net/browse/IRIS-4251

### CODE REVIEW

**Review by macbre — APPROVED** [review]

File: extensions/wikia/WikiFeatures/WikiFeaturesHelper.class.php [hunk 1]
```diff
@@ -41,7 +40,6 @@ class WikiFeaturesHelper extends WikiaModel {
 	// no need to add feature to $release_date if not require "new" flag
 	public static $release_date = array (
 		'wgEnableChat' => '2011-08-01',
-		'wgEnableAchievementsExt' => '2012-02-12',
```
Comment (macbre) [inline extensions/wikia/WikiFeatures/WikiFeaturesHelper.class.php:44]: Please remove these as well:
* https://github.com/Wikia/app/blob/4060af9669c056aa13d01093630fc2e0b7d0b9d0/extensions/wikia/WikiFeatures/images/wgEnableAchievementsExt.png
* https://github.com/Wikia/app/blob/4060af9669c056aa13d01093630fc2e0b7d0b9d0/extensions/wikia/WikiFeatures/WikiFeatures.i18n.php#L45-L46 i18n strings

**Review by macbre — CHANGES_REQUESTED** [review]

File: extensions/wikia/WikiFeatures/WikiFeatures.i18n.php [hunk 2]
```diff
@@ -42,8 +42,6 @@
...
 	'wikifeatures-feature-heading-wgEnableForumExt' => 'Forum',
 	'wikifeatures-feature-description-wgEnableForumExt' => 'Stay on top of every discussion on your wiki by utilizing Wikia\'s new [[w:c:community:Help:Forum|Forum]]. Attract new people to participate in conversations about your wiki\'s content, and keep dedicated wiki editors efficient and engaged with all of the current topics. ',
-	'wikifeatures-feature-heading-wgEnableAchievementsExt' => 'Achievements',
-	'wikifeatures-feature-description-wgEnableAchievementsExt' => '[[w:c:community:Help:Achievements|Achievements]] will help your wiki to grow by incentivizing editors to contribute a variety of content. You can create your own challenges that are unique to your wiki, and even upload your own images for the badges.',
```
Comment (macbre) [inline extensions/wikia/WikiFeatures/WikiFeatures.i18n.php:46]: Let's remove these lines as well :)

```
	'wikifeatures-feature-heading-wgEnableAchievementsExt' => 'Odznaczenia',
```

File: extensions/wikia/WikiFeatures/WikiFeaturesHelper.class.php (hunk 1 above)
Comment (slayful) [reply extensions/wikia/WikiFeatures/WikiFeaturesHelper.class.php:44]: https://wikia-inc.atlassian.net/browse/MAIN-10172 - ticket to upload i18n after this removal

**Comment by slayful** [comment]
@macbre removed the translations as well

**Review by macbre — APPROVED** [review]

**Pull request merged**